app/extensions/auth/oauth2.py

```diff
@@ -155,7 +155,7 @@ def wrapper(*args, **kwargs):
 
                 access_token = None
 
-                log.debug('Using locations = %r' % (locations,))
+                # log.debug('Using locations = %r' % (locations,))
```

Comment:
These lines were coming out a lot in the debug, not adding any value so got removed